src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt

```diff
@@ -43,11 +43,17 @@ class ClientTrackingConfigurationTest {
 
   @Test
   fun shouldAddClientIdAndUserNameToInsightTelemetry() {
+
+    // Given
     val token = jwtAuthHelper.createJwt("bob")
     val req = MockHttpServletRequest()
     req.addHeader(HttpHeaders.AUTHORIZATION, "Bearer $token")
     val res = MockHttpServletResponse()
+
+    // When
```

Comment:
Used the Given/When/Then convention

Reply:
We shouldn't need these comments in the tests, most of the tests should be following this convention

Reply:
The comments are a standard convention with testing!